.github/workflows/avm.platform.publish.tag.yml

```diff
@@ -0,0 +1 @@
+test

```

Comment:
👻

Reply:
@eriqua, this is the file I meant. It does not actually exist. The only reason it shows up here with 'test' was because I tried to get rid of it (which didn't work as it doesn't exist in the branch), then re-created it with the 'test' to delete it again, and now it's in here still as a deleted file with new content.

Reply:
Can we try to delete it from the UI? Need to go away before the pr gets merged

Reply:
Closing and reopening did the trick